schema.ts

```diff
@@ -276,22 +276,6 @@ interface BaseSchema {
    * explain the purpose or contents of any property that is not self-evident.
    */
   description?: string;
-
-  /**
-   * Whether this object schema property is editable by the user in the UI.
-   */
-  /** @hidden */
-  mutable?: boolean;
```

Comment:
@chris-codaio @dweitzman-codaio @huayang-codaio Is moving these a bad idea for some reason I'm not seeing? I think the resulting changes in other repos look sensible to me:
- in `packs`: https://github.com/coda/packs/pull/3692
- in `coda`: https://github.com/coda/coda/pull/95483

Reply:
This doesn't feel unreasonable to me.

Seems like it doesn't break anything interesting in the other repos, and even if we're forgetting about something there's no irreversible harm from shuffling typescript types around.

Reply:
Makes sense.